src/tslint-rules.json

```diff
@@ -108,11 +108,11 @@
 			"call-signature",
 			"parameter",
 			"arrow-parameter",
-			"property-declaration",
-			"variable-declaration",
-			"member-variable-declaration",
-			"object-destructuring",
-			"array-destructuring"
+			"property-declaration"
+//			"variable-declaration",
+//			"member-variable-declaration",
```

Comment:
`member-variable-declaration` should stay such that we explicitly have to typedef the any type, as we can have member variables that are not defined during the declaration, so their type declaration cannot be inferred from the definition.